metadata-ingestion/src/datahub/sdk/_shared.py

```diff
@@ -493,3 +495,86 @@ def domain(self) -> Optional[DomainUrn]:
     def set_domain(self, domain: DomainInputType) -> None:
         domain_urn = DomainUrn.from_string(domain)  # basically a type assertion
         self._set_aspect(models.DomainsClass(domains=[str(domain_urn)]))
+
+
+LinkInputType: TypeAlias = Union[
+    str,
+    Tuple[str, str],  # url, description
+    models.InstitutionalMemoryMetadataClass,
+]
+LinksInputType: TypeAlias = Sequence[LinkInputType]
+
+
+class HasInstitutionalMemory(Entity):
+    __slots__ = ()
+
+    # Internally the aspect is called institutionalMemory, and so much of the code
+    # uses that name. However, the public-facing API is called "links", since
+    # that's what we call these in the UI.
```

Comment:
In GraphQL and OpenAPI we use _InstitutionalMemory_, should we align here with the UI or other APIs? 🤔

Reply:
Because this sdk is for end users, we should align with the UI